### PR TITLE
Try to fix confusing description

### PR DESCRIPTION
**Pull request by MyonKeminta**
The original description of `capacity` may be somewhat confusing. Readers might thinks they can write down a integer and then it's unit is `GB`. However the integer's unit is Byte. The correct representation of GB value is like `"100GB"`.

This issue also occurs in the English document.

### CODE REVIEW

**Comment by lilin90** [comment]
@shenli PTAL

**Comment by wsabc01** [comment]
@liubo0127  Do we need to specify the need to fill out the  unit ？Or give a example .

File: op-guide/ansible-deployment.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ location_labels = ["host"]
         - `rocksdb lockcf block-cache-size(GB)` = MEM * 80% / TiKV 实例数量 * 2.5% (最小 128 MB)
         - `raftdb defaultcf block-cache-size(GB)` = MEM * 80% / TiKV 实例数量 * 2.5% (最小 128 MB)
     2.  如果多个 TiKV 实例部署在同一块物理磁盘上，需要修改 `conf/tikv.yml` 中的 `capacity` 参数:
-        - `capacity` = (DISK - 日志空间) / TiKV 实例数量，单位为 GB
+        - `capacity` = (DISK - 日志空间) / TiKV 实例数量
```
Comment (liubo0127) [inline op-guide/ansible-deployment.md]: We can add an example:
- `capacity` = (DISK - 日志空间) / TiKV 实例数量，例如 "100GB"

**Comment by lilin90** [comment]
@MyonKeminta Thanks! Please address the comment.

**Comment by lilin90** [comment]
@liubo0127 PTAL

**Pull request merged**